app/Http/Controllers/API/TraderController.php

```diff
@@ -158,7 +158,7 @@ public function emailVoucherHistory(Request $request, Trader $trader)
 
         event(new VoucherHistoryEmailRequested(Auth::user(), $trader, $file));
 
-        return response()->json(['message' => '[xXx]The voucher history has been emailed.'], 202);
+        return response()->json(['message' => '[xXx]Thanks. If you don\'t receive an email with your voucher history, please try again later.'], 202);
```

Comment:
Also note that @alexalala is working on a refactor of error messages at the moment.